config/config.go

```diff
@@ -122,6 +123,12 @@ var (
 		MaxBackoff: model.Duration(100 * time.Millisecond),
 	}
 
+	// DefaultMetadataConfig is the default metadata configuration for a remote write endpoint.
+	DefaultMetadataConfig = MetadataConfig{
+		Send:         true,
+		SendInterval: model.Duration(15 * time.Second),
```

Comment:
Should this be once a minute as that is the default scrape interval in Prometheus?